docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,498 @@
+          parent: { this: {} },
+          viewer: {
+            union: {
+              child: [
+                { this: {} },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: { directly_related_user_types: [{ type: 'user' }] },
+          },
+        },
+      },
+      {
+        type: 'document',
+        relations: {
+          parent: { this: {} },
+          viewer: { this: {} },
+          can_read: {
+            union: {
+              child: [
+                { computedUserset: { object: '', relation: 'viewer' } },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: { directly_related_user_types: [{ type: 'user' }] },
+            can_read: { directly_related_user_types: [] },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+After making these changes, <ProductName format={ProductNameFormat.ShortForm}/> will start validating the tuples more strictly, for example, you won’t be able to assign a `document` as a member of a `group`. If your application is writing invalid tuples, you’ll start getting errors when invoking the [`Write` API](/api/service#/Relationship%20Tuples/Write  ).
+
+## Disallowing String Literals in user_ids
+
+With version 1.0 models, you could write a tuple where the user id did not specify a type, for example:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      _description: '"bob" is a member of the "sales" group',
+      user: 'bob',
+      relation: 'member',
+      object: 'group:sales',
+    },
+  ]}
+/>
+
+However, with version 1.1 you always need to specify an object, so “bob’” is no longer a valid identifier. If you don’t have a type in your model that defines relations for users, you can add a ‘user’ type with no relations to your model, for example:
+
+<AuthzModelSnippetViewer
+  configuration={{
+    type_definitions: [{ type: 'user', relations: {} }],
+  }}
+/>
+
+You can then use that type when writing tuples:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:bob',
+      relation: 'member',
+      object: 'group:sales',
+      _description: '"user:bob" is a member of the "sales" group',
+    },
+  ]}
+/>
+
+## Enforcing Userset Type Restrictions
```

Comment:
Drop the the word "Userset" in this definition. It's a little misleading. Instead just say "Enforcing Type Restrictions"

Reply:
Done